src/components/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -693,6 +703,10 @@ $grid-spacing: var(--kui-space-30, $kui-space-30);
 
 // v-calendar overrides
 .k-datetime-picker {
+  // All calendar dates use v-calendar's small text variable
+  // stylelint-disable-next-line
+  --vc-text-sm: var(--kui-font-size-20, 12px);
```

Comment:
@adamdehaven if this is not okay, I can override individual class names instead. 

This seemed like the simplest method, otherwise I'd write multiple lines of CSS to achieve the same effect.

Reply:
Works for me, but can you use this instead?
```suggestion
  --vc-text-sm: var(--kui-font-size-20, #{$kui-font-size-20});
```

Reply:
Ah, i need to wrap the var in `#{}`... thanks for that. 

Updated that line, and added `--vc-font-family` as well.
<img width="348" alt="Screenshot 2023-12-15 at 12 45 46 PM" src="https://github.com/Kong/kongponents/assets/103061463/9bee8794-4fec-45b8-91ae-618ad2657813">